web-server/v0.4/src/pages/Search/SearchList.js

```diff
@@ -140,7 +140,7 @@ export default class SearchList extends Component {
 
     dispatch({
       type: 'dashboard/updateSelectedController',
-      payload: params['run.controller'],
+      payload: params[0]['run.controller'],
```

Comment:
Can we also rename `params` to `selectedResults`?

Reply:
I've updated the reference here.